content/en/docs/getting-started/_index.md

```diff
@@ -74,15 +95,19 @@ sudo systemctl enable microshift --now
 {{< /tabs >}}
 <br/>
 
-To access the cluster, install the OpenShift client or kubectl if you haven't done so yet:
+### Install Clients
+
+To access the cluster, install the OpenShift client or kubectl.
 
 ```Bash
 curl -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz
 tar -xzvf oc.tar.gz
 sudo install -t /usr/local/bin {kubectl,oc}
 ```
 
-Depending on the user that will be administrating the system it may be required to copy the kubeconfig to a location that can be accessed by the user.
+### Copy or Set Kubeconfig
```

Comment:
We're only describing the "copy", so "Copy Kubeconfig"?

Reply:
done